winequality.ipynb

```diff
@@ -0,0 +1,1386 @@
TPwradu5jw9TcYU1P5X6eO9O35isXypKQkPhozjmOhoRRxc+fLT8ZSwrc4q4LWMH/R4rS4EydP8euCeVSqUIEpM79lxV9BXLt+nT2b1lu7SNmyY8cOJk6cSGpqKp07d6ZXr14Wy/fv389XX33FyZMn+fTTT2ndunXasoEDB3L48GFq1arFN998Y+XMs+de5UtKSmLUqFGEhITg7u7OZ599hq+vLykpKYwbN47jx49jNBpp3749vXv3BuCnn35i6dKlAHTu3Jnu3btbu1g5MvTZlgRWKEPsjQR6z/gpr9PJVTl5P+c3e3buYNrXprI+06kz3Xv2sli+ZPFCVq9Yjr29Pe5FizJk+EiKFS+eN8k+gP27dzFn2mRSjak81b4DXV+2/E5e/uvPrP1zlal8RYowcOgwDMWKARARdpVpX04gMjwclGLk51/iY6Nl11oz6fsf2bH/IE4FCzJiQD8qlSubIS7k1BnGTf+WW0lJNKpTi/d6v4JSihNnzzFh9vckJCZS3ODNmLffxNXZOQ9Kkrnte/byxbSZ5uuIdrzWvZvFctN1xJeEnPgXd7fCTBg1nBLFihEbd433R4/j6PFQOrVrw7B33gLgxs2b9H77vbT1wyMieeapVgx9602rlutetNbMWreVvafOUdDBkcEdW1G+mHeGuI9/Xkl0/A2MqalUK+lL/7bNsLcztRkt33uIlX8fxt5O0aB8Gfq0bGTtYmSL1popS5az++hxCjo6Mqzni1Qo5WcRk5iUxKg5P3I5Mgo7OzsaVa/C/3V+BoB//j3N1N9WcPrSFUa+9jIt6tTIi2KIR9Rj2QKrlJqvlOqayfwySqkj97ktX6XUb1ks26yUqvegeT4MRqORzyZPZfqE8fyx4DuCNm7i1NlzFjFLV/+Fm2thVi7+gR5dn2Py7DkAFHV3Y/L4cfz2/VzGfTiU4eM/T1vni9Ej+PW72fz+/Vxi4mJZt3mLVcuVGaPRyKcTv2LG11+x/KdF/LV2PafOnLGI+WPFKtzcCrP6t1955aUX+Xr6DAA6tGvLbz8u4LcfFzB+1Eh8ixenUoUKADRv2pif5s2xenmyy2g0MmHCBKZMmcKSJUtYs2YNp0+ftogpVqwYo0ePpm3bthnWf+WVVxg7dqy10r1v2Snf8uXLKVy4MMuWLaN79+5MnToVgPXr15OUlMQvv/zCwoUL+eOPP7h8+TInT55k6dKl/PDDDyxevJht27Zx/vz5vCjeAws6eJyhC1fmdRq5Lqfv5/zEaDQyeeIEPv96Ct//tISNa9dw9oxlWctXrMTM+T8yd9HPNHuyFbOnTcmjbO+f0Wjk28mTGDVhItMWLGTrxvWcP2v5nVw2oAKTvp3LlHkLaNS8BfO/nZG27Jvxn9ClW3em/7CIiTNnU6RoUWsXIdt2HPiHC1eu8tvUr/jw//rwxZz5mcZ9Med7hv1fH36b+hUXrlxl58FDAIyfNZcBL7/I4kmf07xBPRau+NOK2d+d6TpiGtM//5Q/5s8haMPmTK4jgnAr7MrKRfPp8fz/mPztdwAULODIgNde5b03+1nEuzg78+vcWWn/ivsYaNW0sbWKlG17T53jcnQc373Rg7efacG0oM2Zxg3r0pYZfbsx6/WXiLuZwNaQUwD8c/Yiu/49w4y+3fi2X3eea1jLitnfn91Hj3MxPJJFoz/g/Ze7MunnPzKNe7F1c34cNZS5w97l8Kmz7Dp6HACDRxGGvfICrerZbhlF/vVYVmBzi1LKQWt9WWudoTJsq44cD6VkCV/8fH1xdHSkbcsWbN6+3SJm8/YddGzXBoDWzZux5+8DaK2pFBCAwcsLAP+yZUhKSiIpKQkAVxcXAFKMRpKTU7CF7vuHj4VQys+PkiVK4OjoyNNPtWLTlq0WMZu2bqXTM6a7hU892YLd+/5Ga20R89e6dTzz1O0WnZrVquFtfh1s0dGjRylZsiR+fn44OjrSpk0bgoODLWJ8fX0JCAjAzi7jV0CDBg1wtqE7/XfKTvmCg4Pp0KEDAK1atWLPnj1pxzUxMZGUlBQSExNxdHTExcWFs2fPUr16dZycnHBwcKBOnTps2rTJ6mXLiUPnLnM9ITGv08h1OX0/5yfHjx2lhF9JfEuYytryqTbs2GJZ1tp16+Hk5ARAlWrViAgPy4tUH8i/x0MoVsKPYr6m7+SmLVuzZ/s2i5gatetQ0Fy+ilWqEhURAcD5s2cwGo3UqlcfgELOzmlxtmjL3r95unkTlFJUr1Ce6zduEBkTYxETGRPDjYQEqlcMQCnF082bELxnHwDnLl+hdpVKADSsUY1Nu/ZavQxZOXI8lJK+vvj5FjdfRzRn8/YdFjGbt++kY9unAPN1xH7TdUShQoWoXb0aBQoUyHL75y5eIjo2ljo1qj/UcjyIXSfO0Kp6RZRSVC5RjPjEJKLjb2SIcyloKp8xNZVkY2raNdGf+4/wQmAdCjiYeqgVcbHdc+22Q0dp27AuSimqli1N/M1EouKuWcQ4FShAnYrlAXB0cKBCyRJExMQBUNzTA38/X+zsbOCCMJ+xt1M2889W5e+zfTYppXoqpQ4ppf5RSv1ont1MKbVDKXU6i9ZYJ6XU90qpw0qpA0qpJ83zeymlliilVgJr07faKqUKKaV+Nu/rF6BQuu21UUrtVErtN6/vap7/uVLqmHmdiQ/7tQiPiKSYtyFt2sfbm/CIqDtioijmbeoS4+Bgj6urC7F3fGmtD95KpfLlLU5Cbw75gJadu+LsXIjWzZs9xFJkT3hEBMUM6cpqMBBmvhiyiPExxTg4OJjLGmcRE7R+A0+3eerhJ5xLwsPD8fHxSZs2GAyEh4fnYUa5KzvlSx9jOq6uxMXF0bp1a5ycnGjXrh0dOnSgR48euLu74+/vz4EDB4iNjSUxMZHt27cTFpZ/KgaPskf9/ZxeZEQ4BsPtsnoZDEREZF3W1SuX0yDQNrsfZiYqIgKvdOcfT2/vtApqZtb9uYq6DRoCcPnCBVxcC/PZiI94t29vvp85Ha
```

Comment:
Please add a comment indicating how you decided on this train-test split ratio. We are asking for this input at this point because one of goals of the project is to better understand how this choice influences the outcomes of the model.

Reply:
Thank you very much for your feedback @dzeber. I will update it right away.